documentstore/domain.py

```diff
@@ -824,6 +824,10 @@ def insert_issue(self, index: int, issue: str) -> None:
     def remove_issue(self, issue: str) -> None:
         self.manifest = BundleManifest.remove_item(self._manifest, issue)
 
+    @property
+    def issues(self):
+        return self.manifest.get("items", [])
```

Comment:
Eu acho que devia aqui ter tipo in `BundleManifest.get_item`, mas vi que não tem esse metodo implememtado e tambem vi que no treço abaixo (`domain.BundleManifest`) acontece ate um caso pior, onde pode ter ate erro

https://github.com/scieloorg/document-store/blob/837a9c8f96ca18c2e7d71abbc73a6e1c137922e7/documentstore/domain.py#L514-L515

Pois como todos os outro `get`, `set` tem esse metodo estatico para manipular os dados vindo do manifest, acho legal manter o mesmo padrão

mais isso tambem pode ser um caso de uma futura re-fatoração também

Reply:
Boa observação @cesarbruschetta, eu optei por utilizar o `.get` para evitar possíveis problemas de `KeyError` todavia o acesso `self.manifest` é o item de `cópia` do confere?

Só para clarificar o comentário, sua sugestão é editar o PR?

Reply:
@joffilyfe, O `BundleManifest.new` já garante que o `items` com uma lista vazia:

https://github.com/scieloorg/document-store/blob/master/documentstore/domain.py#L334-L343

O uso correto do BundleManifest garante que o `KeyError` não ocorra